iroha/src/block_sync.rs

```diff
@@ -203,13 +209,13 @@ pub mod message {
                                 .send_to(peer)
                                 .await
                         {
-                            log::warn!("Failed to request blocks: {:?}", err)
+                            warn!("Failed to request blocks: {:?}", err)
```

Comment:
There is some inconsistency in the usage style of macros, you either use a raw `warn` or `iroha_logger::warn` in different places. Please consider using them everywhere in the same way, according to our codebase style `iroha_logger::warn` is prefered.